website_and_docs/content/blog/2023/java-8-support.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Java 8 support in Selenium"
+linkTitle: "Java 8 support in Selenium"
+date: 2023-06-09
+tags: ["selenium", "java"]
+categories: ["releases"]
+author: Diego Molina ([@diegofmolina](https://twitter.com/diegofmolina))
+description: >
+   On September 30, 2023, Java 11 will be the minimum version supported by Selenium.
+---
+
+
+“If it ain’t broke, don’t fix it” is a saying you may have heard, but sometimes 
+it’s necessary to move on from our old favorites. That’s why we’re announcing that
+**Selenium will stop supporting Java 8 on September 30, 2023**. This applies for
+both the Java bindings and the Selenium Grid.
+
+Selenium has long supported Java 8, but as technology evolves, so must we. One of 
+the primary reasons for this change is that Java 8 reached the end of active support 
+[over a year ago](https://endoflife.date/java). In addition, a 
+[bug](https://github.com/SeleniumHQ/selenium/issues/9528) while using a third party Java 8 
+HTTP client motivated us to [implement a Java 11 HTTP client](blog/2022/using-java11-httpclient/).
```

Comment:
I wanted to write a replacement for this section, but realized I might not have it correct. If someone is using Java 17, they aren't using the Java 11 client, right? It's just that the features we need don't exist in the default client until v11? This section seems either too much detail or not enough if people don't already understand our Http client.

Reply:
The Java 11 client is a direct opt in.

Reply:
Does not matter the Java version being used.

Reply:
The text is just to show reasoning. So what is the proposal? I'd rather get this out soon.

Reply:
I think it is confusing. If I'm using Java 17, what is this about Java 11? Would it be more accurate to say that we are switching to the native Java HTTP Client, and it didn't have the features required for Selenium until Java 11?

Reply:
OK, so what is the suggestion?

Reply:
All in all, this was just to add a bit of more reasoning. If we leave it out, I am also fine with it. I would just prefer to get the post out.

Reply:
How about:

> In addition, our default HTTP Client has not had a major release in several years, and a [bug](https://github.com/SeleniumHQ/selenium/issues/9528) has been found that we can not fix. We have decided to [move to the native Java HTTP Client](blog/2022/using-java11-httpclient/), but it requires using Java 11 or greater. The sooner we make this change, the sooner we can avoid dealing with this issue.

Reply:
Sounds good! 
Let me change that right away.

Reply:
Pushed changes. I will resolve this one and if we want to check it again, we can reopen.